templates/fastd/fastd_nodes01.conf

```diff
@@ -8,10 +8,10 @@ method "salsa2012+umac";
 method "null+salsa2012+umac";
 method "null";
 mode tap;
-interface "fastd_nodes";
+interface "fastd_nodes01";
```

Comment:
use vars in the template - see https://docs.ansible.com/ansible-container/container_yml/template.html